torch/csrc/jit/fuser/compiler.cpp

```diff
@@ -86,11 +87,13 @@ static const Node* usedInFusedChunk(const Value* input) {
 static void setInputChunkDescriptors(KernelSpec& spec) {
   spec.inputChunks().reserve((spec.graph())->inputs().size());
   for (const Value* input : (spec.graph())->inputs()) {
-    if (const Node* chunk = usedInFusedChunk(input)) {
-      spec.inputChunks().emplace_back(
-          chunk->i(attr::chunks), chunk->i(attr::dim));
-    } else {
-      spec.inputChunks().emplace_back(1, 0);
+    if (input->type()->isSubtypeOf(DynamicType::get())) {
```

Comment:
nit: 
```
if (!input->type()->isSubtypeOf(DynamicType::get())) {
   continue;
}
```
keeps the blame

Reply:
I must admit that `continue` is very foreign to me (feels like goto), but that's me and I'll adapt.